source/class.cpp

```diff
@@ -918,6 +924,7 @@ string bind_constructor(ConstructorBindingInfo const &CBI, uint args_to_bind, bo
 
 		if( CBI.C->isAbstract() ) c = fmt::format(constructor_template, params, args.second, CBI.trampoline_qualified_name);
 		else if( CBI.trampoline ) c = fmt::format(constructor_with_trampoline_template, params, args.second, CBI.class_qualified_name, CBI.trampoline_qualified_name);
+		else if( args_to_bind )c = fmt::format(constructor_template_with_pyArg, params, args.second, CBI.class_qualified_name, argsHelper);
 		else c = fmt::format(constructor_template, params, args.second, CBI.class_qualified_name);
```

Comment:
is this case became unreachable? And if so then `constructor_template` will need to be commented out.

Reply:
I do not think it is going to be unreachable. For a case that all constructors have default values, the last case is reachable.

Reply:
Done. Thank you.